library/src/scala/runtime/stdLibPatches/Predef.scala

```diff
@@ -61,4 +61,5 @@ object Predef:
     inline def ne(inline y: AnyRef | Null): Boolean =
       !(x eq y)
 
+  extension (inline opt: Option.type) inline def fromNullable[T](t: T|Null): Option[T] = Option(t).asInstanceOf
```

Comment:
Does this need a `@experimental` or `@since` annotation?

Reply:
Added an experimental annotation to `fromNullable`